src/scoreboard/components/Scoreboard/Scoreboard.component.tsx

```diff
@@ -153,23 +156,35 @@ export const Scoreboard: FC<ScoreboardProps> = () => {
     const average = formatDivision(playerStats[player].total, playerStats[player].roundsPlayed, 1);
     const stdDev = standardDeviation(playerScores, average);
 
+    const scoresRev = [...playerScores].reverse();
+    const scoreLeft = playerStats[player].remaining
     // Determine a reaction
-    if (stdDev > 40) {
+    if (stdDev > 40 && scoresRev[0] < scoresRev[1] && scoreLeft > CLOSE_BREAKPOINT) {
       showReaction(lawOfAverages);
     }
+    if ((_throws?.length ?? 0) > 0) {
+      if (_throws?.includes(20) && _throws?.includes(5) && _throws?.includes(1)) {
+        showReaction(bagOfDicks);
+      }
+      const lowestValue = _throws?.sort()?.[0]
+      if (lowestValue && _throws?.includes(lowestValue*2) && _throws?.includes(lowestValue*3)) {
```

Comment:
Technically this is a bit too lax as it would trigger for a single 1, single 2, and single 3 (for example).  There may be something in the `DartboardClickDetails` that denotes a single vs double vs triple, but I can't remember.